src/bin/main.rs

```diff
@@ -375,6 +385,27 @@ fn sender_print_code(term: &mut Term, code: &magic_wormhole::Code) -> eyre::Resu
     Ok(())
 }
 
+fn execute_shell_command(
+    term: &mut Term,
+    matches: &clap::ArgMatches<'_>,
+    code: &magic_wormhole::Code
+) -> eyre::Result<()> {
+    if let Some(shell_command) = matches.value_of_os("shell-command") {
+        let mut child = std::process::Command::new("/bin/sh")
```

Comment:
```suggestion
        let mut child = std::process::Command::new("sh")
```

Don't hard-code the absolute path. Alternatively, `/usr/bin/env sh` should work fine as well.

Reply:
I guess `Command` handles the path search, so just `sh` is fine. Should I also include a Windows if/else that uses `cmd /C`?

Reply:
Ugh, Windows support. To be honest I don't really care, so try to make it work, otherwise simply gate that option behind a feature flag (something like `#[cfg(not(target_os = "windows"))]` IIRC, but I recommend reading [the documentation on this](https://doc.rust-lang.org/reference/conditional-compilation.html)).